README.md

```diff
@@ -1,205 +1,4 @@
-## An OS X GUI for Appium
-
-[![Build Status](https://travis-ci.org/appium/appium-dot-app.svg?branch=master)](https://travis-ci.org/appium/appium-dot-app)
-
-If you are new to Appium then please see the [Getting started](http://appium.io/getting-started.html) guide for more information
-about the project.
-
-To install:
-
-1. Download the [latest version](https://bitbucket.org/appium/appium.app/downloads/appium.dmg) from [Appium.io](http://appium.io/).
-2. Mount the disk image.
-3. Drag Appium.app to your Applications folder.
-
-### Build with XCode
-
-Install CocoaPods:
-
-1. Open Termianl
-2. `sudo gem install cocoapods`
-3. `pod setup`
-4. `cd appium-dot-app`
-5. `pod install`
-
-To Open in XCode: 
-
-open the `appium-dot-app/Appium.xcworkspace` project file
```

Comment:
also, if we are deprecating this and the dot.exe, we should change the appium.io webpage to point to this new repo/executable instead of the old dot.exe and dot.app.